dev/tasks/vcpkg-tests/github.windows.yml

```diff
@@ -73,6 +73,14 @@ jobs:
           `vcpkg fetch nuget | tail -n 1` \
             setapikey "$GITHUB_TOKEN" \
             -source "https://nuget.pkg.github.com/$GITHUB_REPOSITORY_OWNER/index.json"
+      - name: Cache nasm
+        run: |
+          # echo "VCPKG_ROOT=$env:VCPKG_INSTALLATION_ROOT" | Out-File -FilePath $env:GITHUB_ENV -Append
+          New-Item -ItemType "directory" -Path "C:\vcpkg\downloads" -Force
+          Invoke-WebRequest -URI "$env:BASE_URL/$env:FILE_NAME" -OutFile "C:\vcpkg\downloads\$env:FILE_NAME"
+        env:
+          BASE_URL: https://github.com/microsoft/vcpkg/files/12073957
+          FILE_NAME: nasm-2.16.01-win64.zip
```

Comment:
@assignUser any idea how I could cache this for vcpkg?

Reply:
hm I think creating the downloads folder in `$GITHUB_WORKSPACE` ~~and using github.com/actions/cache to cache that~~ (no normal caching in crossbow...) in combination with the download script you have should work + maybe a check for `$GITHUB_ACTIONS` in *vcpkg.bat to avoid adding that option when not in gha (probably not used by anyone locally but how knows...)?

Reply:
I am surprised that it is not being cached already through nuget...